src/main/java/org/opensearch/security/tools/PasswordSetup.java

```diff
@@ -0,0 +1,680 @@
+			} catch (IOException | CertificateException e) {
+				throw new IllegalArgumentException("Error while reading certificate file " + caCertFile, e);
+			}
+		}
+
+        if (cert != null && key != null) {
+			File certFile = Paths.get(cert).toFile();
+			X509Certificate[] certificates;
+			PrivateKey privateKey;
+			try (FileInputStream in = new FileInputStream(certFile)) {
+				certificates = PemKeyReader.loadCertificatesFromStream(in);
+			} catch (FileNotFoundException e) {
+				throw new IllegalArgumentException("Could not find certificate file " + certFile, e);
+			} catch (IOException | CertificateException e) {
+				throw new IllegalArgumentException("Error while reading certificate file " + certFile, e);
+			}
+
+			File keyFile = Paths.get(key).toFile();
+			try (FileInputStream in = new FileInputStream(keyFile)) {
+				privateKey = PemKeyReader.toPrivateKey(in, keypass);
+			} catch (FileNotFoundException e) {
+				throw new IllegalArgumentException("Could not find certificate key file " + keyFile, e);
+			} catch (IOException e) {
+				throw new IllegalArgumentException("Error while reading certificate key file " + keyFile, e);
+			}
+
+			String alias = "al";
+			KeyStore keyStore = PemKeyReader.toKeystore(alias, "changeit".toCharArray(), certificates, privateKey);
+            sslContextBuilder.loadKeyMaterial(keyStore, "changeit".toCharArray(), (aliases, socket) -> alias);
+		}
+
+		return sslContextBuilder.build();
+	}
+
+    private static int issueWarnings(RestHighLevelClient restHighLevelClient) throws IOException {
+		Response res = restHighLevelClient.getLowLevelClient().performRequest(new Request("GET", "/_nodes"));
+
+		if (res.getStatusLine().getStatusCode() != 200) {
+			System.out.println("Unable to get nodes " + res.getStatusLine());
+			return -1;
+		}
+
+		JsonNode resNode = DefaultObjectMapper.objectMapper.readTree(res.getEntity().getContent());
+
+		int nodeCount = Iterators.size(resNode.at("/nodes").iterator());
+
+		if (nodeCount > 0) {
+
+			JsonNode[] nodeVersions = Iterators.toArray(resNode.at("/nodes").iterator(), JsonNode.class);
+
+			Version maxVersion = Version.fromString(Arrays.stream(nodeVersions).max((n1, n2) -> Version.fromString(n1.asText()).compareTo(Version.fromString(n2.asText()))).get().asText());
+			Version minVersion = Version.fromString(Arrays.stream(nodeVersions).min((n1, n2) -> Version.fromString(n1.asText()).compareTo(Version.fromString(n2.asText()))).get().asText());
+
+        if(!maxVersion.equals(minVersion)) {
+            System.out.println("ERR: Your cluster consists of different node versions. It is not allowed to run securityadmin against a mixed cluster.");
+            System.out.println("         Minimum node version is "+minVersion.toString());
+            System.out.println("         Maximum node version is "+maxVersion.toString());
+            if(!ALLOW_MIXED) {
+                return -1;
+            }
```

Comment:
Seems like a left over from another script. Don't think we need this, do we?

Reply:
I don't think so - I'll remove it